bindings/python/build_defs.oss.bzl

```diff
@@ -77,6 +77,125 @@ def iree_py_test(**kwargs):
     #   imports
     py_test(legacy_create_init = False, **kwargs)
 
+def iree_py_test_suite(
+        name,
+        srcs,
+        excludes = [],
+        deps = [],
```

Comment:
Turns out that it's better to use "None" here rather than an empty list. You already do this for `size` and `tags`, which is good. We should update our other macros to follow this convention. https://docs.bazel.build/versions/master/skylark/macros.html#conventions

Reply:
noted! `py_test` actually doesn't work if you set a default for `tags` in a macro, and the behavior/error message ("trying to mutate a frozen object") seemed pretty odd to me coming from python.

Reply:
wat :confused:

Reply:
I figured it out by guessing and checking to see where exactly it broke 🙃